src/parser.ts

```diff
@@ -147,7 +147,7 @@ function parseNonLiteral(
         comment: schema.description,
         deprecated: schema.deprecated,
         keyName,
-        standaloneName: standaloneName(schema, keyNameFromDefinition, usedNames),
+        standaloneName: standaloneName(schema, keyNameFromDefinition, usedNames, options),
```

Comment:
let's update unit tests for that to keep lib in health state

Reply:
Could you explain what you mean by this? I don't see the unit tests -- just end to end?